openbas-front/src/admin/components/agents/Agents.tsx

```diff
@@ -84,6 +82,7 @@ const Executors = () => {
             <ExecutorSelector
               executor={executor}
               setSelectedExecutor={setSelectedExecutor}
+              isEEExecutor={executor.executor_type == OPENBAS_TANIUM || executor.executor_type == OPENBAS_CROWDSTRIKE}
```

Comment:
question (for the whole PR): shouldn't we add this as a more generic property somewhere ? Maybe we need to add a isEE propetry in all executors in addition to the executor_type ?